### PR TITLE
couper.version in example

### PR DESCRIPTION
**Pull request by afflerbach**

### CODE REVIEW

File: static-responses/README.md [hunk 1]
```diff
@@ -104,7 +106,8 @@ $ curl localhost:8080/app/conf
 {
   "debug": true,
   "env": "local",
-  "version": "1.0"
+  "version": "1.0",
+  "couper": "1.3.1"
```
Comment (johakoch) [inline static-responses/README.md:110]: `couper` will probably be the first key.

File: static-responses/README.md [hunk 2]
```diff
@@ -85,6 +86,7 @@ server "responses" {
         version = env.APP_VERSION
         env = env.APP_ENV
         debug = env.APP_DEBUG == "true"
+        couper = couper.version
```
Comment (johakoch) [inline static-responses/README.md:89]: Add this line to `couper.hcl`?

**Review by johakoch — CHANGES_REQUESTED** [review]

File: static-responses/README.md (hunk 1 above)
Comment (afflerbach) [reply static-responses/README.md:110]: :heavy_check_mark:

File: static-responses/README.md (hunk 2 above)
Comment (afflerbach) [reply static-responses/README.md:89]: Sure!

**Review by johakoch — APPROVED** [review]

**Pull request merged**